### DISCUSSION TITLE
How to replace specs-actors.

### DISCUSSION THREAD

**Discussion by dafsic** [filecoin-project/lotus#5739] [General Q&A] [answered]
I added some log in specs-actors, and i added 'replace github.com/filecoin-project/specs-actors/v3 => ../specs-actors' in go.mod of lotus. But it is invalid, the log not print out.

**Comment by dd45e640b42e6da7da96faee3996ef7c** [comment, marked as answer]
it gives you a compile error?

**Reply by dafsic** [reply]
sorry, there is a problem with my code, but this method is right.
